src/CMakeLists.txt

```diff
@@ -407,6 +421,16 @@ if(USE_VARIABLE_SPLASH_SCREEN)
   target_compile_definitions(mudlet PRIVATE INCLUDE_VARIABLE_SPLASH_SCREEN)
 endif()
 
+if(WIN32)
```

Comment:
Why is this block extra and not combined with the `WIN32` block below?

Reply:
Done.